packages/babel-plugin-emotion/src/utils/transform-expression-with-styles.js

```diff
@@ -7,6 +7,21 @@ import { getSourceMap } from './source-maps'
 import { simplifyObject } from './object-to-string'
 import { appendStringToExpressions, joinStringLiterals } from './strings'
 
+function createSourceMapConditional(t, production, development) {
+  return t.conditionalExpression(
```

Comment:
maybe use a babel template for readability?
```js
  const tmpl = template(
    'process.env.NODE_ENV !== "production" ? DEV_VALUE : PROD_VALUE',
    {placeholderPattern: /(DEV_VALUE)|(PROD_VALUE)/},
  )
```

Reply:
tbh, i think this is readable enough and i'm not a fan of the relying on casing by default of the babel template api so I'd rather have it like this than have to look up the api of babel template to understand what everything does. I doubt this will change much anyway so, I think it's fine how it is.